test/integration/pkg-put-write-Integrity.js

```diff
@@ -1,116 +0,0 @@
-'use strict';
```

Comment:
Why did you rename this file to have a capital I?

Reply:
Its renamed to not have a capital I.